vassal-app/src/main/java/VASSAL/build/module/map/boardPicker/board/mapgrid/PolygonEditor.java

```diff
@@ -373,6 +453,64 @@ else if (SwingUtils.isContextMouseButtonDown(e)) {
         insertVertex(polygon, ins, e.getX(), e.getY());
         selected = ins;
         repaint();
+
+        if (myConfigurer != null) {
+          myConfigurer.updateCoords(polygon);
+          if (selected >= 0) {
+            myConfigurer.updateCoord(polygon.xpoints[selected], polygon.ypoints[selected]);
+          }
+          else {
+            myConfigurer.updateCoord("");
+          }
+        }
```

Comment:
Didn't you define a function to do this up above?

Reply:
good point - cleaned up a bunch of that stuff